frontend/src/app/event/widgets/event-users-list/event-users-list.widget.html

```diff
@@ -0,0 +1,44 @@
+<mat-card class="registrations-list" appearance="outlined">
+  <div class="registrations-title-container">
+    <div class="registrations-title">
+      <mat-card-title>Registrations</mat-card-title>
+    </div>
+
+    <!-- Buttons -->
+    <!-- Placeholder template for future registration actions (mass email reminders, delete all, etc.) -->
+    <div class="registrations-actions">
+      <button mat-icon-button>
+        <mat-icon>mail</mat-icon>
+      </button>
+    </div>
```

Comment:
We should remove this for production purposes; clicking on the email icon was so tempting and ultimately sad. 😭 

I'll do this in my cleanup commit.

Reply:
Pushed this fix.